### PR TITLE
Initial sound work

### PR DESCRIPTION
**Pull request by g7**

### CODE REVIEW

**Comment by g7** [comment]
Note: this commit needs to be reverted:

https://github.com/latte-dev/android_kernel_xiaomi_latte/commit/f6a618c81e3f555c69dd193c0de6e614bf4421c4

**Comment by g7** [comment]
If pulseaudio fails to start, remove the bluez configurations from `/etc/pulse/xpolicy.conf.d/` and touch a dummy file there (had trouble with starting pulseaudio with an empty `xpolicy.conf.d` directory).

**Comment by g7** [comment]
Squashed and rebased, everything should be good from my end.

File: sparse/etc/pulse/ia32_rt5659_soundcard.pa [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+### stored in /usr/share/application
+#load-module module-augment-properties
+
+load-module module-null-sink sink_name=sink.null rate=48000
+
+load-module module-alsa-card device_id=2 card_name=card.rt5659 sink_name=sink.rt5659 source_name=source.rt5659 rate=48000
+
+load-module module-null-sink sink_name=sink.fake.sco rate=8000 channels=1
+load-module module-null-source source_name=source.fake.sco rate=8000 channels=1
+load-module module-bluez4-discover sco_sink=sink.fake.sco sco_source=source.fake.sco
```
Comment (piggz) [inline sparse/etc/pulse/ia32_rt5659_soundcard.pa]: This line needs to be
load-module module-bluetooth-discover bluez4_args="sco_sink=sink.fake.sco sco_source=source.fake.sco"

**Review by piggz — COMMENTED** [review]
With one fix it works for me

**Comment by g7** [comment]
Should be fixed now.

**Pull request merged**